DRAFT: Multi-Party Payout.md

```diff
@@ -0,0 +1,204 @@
+
+  * Define new Operator
+  * Define payout to Operator
+
+### Problems with the previous system
+
+While elegant, under the current system, the masternode reward is paid to the operator based on the `<operatorReward>` field and any difference from 100% is automatically paid to the owner address.
+
+The limitation of rewards paid to only 2 addresses based on a single value field prevents automatic/trustless sharing of the rewards beyond the owner and/or operator, a critical element for enabling trustless shares in a masternode and introducing Dash native DeFi opportunities through staking (among other use cases).
+
+## Prior Work
+
+* [DIP 0002: Special Transactions](https://github.com/dashpay/dips/blob/master/dip-0002.md)
+* [DIP 0003: Deterministic Masternode Lists](https://github.com/dashpay/dips/blob/master/dip-0003.md)
+
+## New Stake Holder Reward Fields within ProRegTx and ProUpRegTx
+
+In the new system the ProRegTx and ProUpRegTx (DIP3) may include an additional fields to define additional parties to receive rewards beyond the owner and operator and the % of owner rewards each party receives.
+
+This DIP defines:
+
+  1. the sharedReward field
+  2. the shareHolderReward field
+  3. corresponding update to the ProRegTx
+  4. corresponding update to the ProUpRegTx
+
+All other transactions, transaction fields, the mechanisms for adding or removing masternodes and defining MN quorum and valid subset remain unchanged.
+
+## Registering a Masternode (ProRegTx)
+
+To join the masternode list, masternode owners must submit a special transaction (DIP2) to the network. DIP 3 defines the ProRegTx and the ProRegUpTx message, parts, and payload: `<magicString><payoutStr>|<operatorReward>|<ownerKeyAddress>|<votingKeyAddress>|<payloadHash>`
+
+We propose adding an additional part (`<sharedReward>`)to produce a message that includes:
+
+`<magicString><payoutStr>|<operatorReward>|<ownerKeyAddress>|<votingKeyAddress>|<sharedReward>|<payloadHash>`
+
+The individual parts of the message are:
+
+1. `<magicString>`: (DIP3) A fixed string which is always "DarkCoin Signed Message:" with a newline character appended.
+Please note that existing tools (e.g. the RPC command `signmessage`) usually add the magicString internally, making it
+unnecessary to manually add it.
+2. `<payoutStr>`: (DIP3) The Dash address corresponding to the scriptPayout field of the ProRegTx. In case scriptPayout is not a
+P2PK/P2PKH/P2SH script, `<payoutStr>` must be set to the hex representation of the full script.
+3. `<operatorReward>`: (DIP3) The operatorReward field of the ProRegTx.
+4. `<ownerKeyAddress>`: (DIP3) The Dash address corresponding to the keyIdOwner field of the ProRegTx.
+5. `<votingKeyAddress>`: (DIP3) The Dash address corresponding to the keyIdVoting field of the ProRegTx.
+6. `<sharedReward>`: (NEW) the sharedReward field of the ProRegTx.

```

Comment:
would add more context for `sharedReward` - expand on the descriptor, such as the type